### PR TITLE
prepare to publish convert

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by jakemac53 — APPROVED** [review]

**Comment by jakemac53** [comment]
@kevmoo not sure if there were any actual breaking changes we want to land here though (before publishing)

**Comment by natebosch** [comment]
There are no issues with `next-breaking-release` and no deprecated annotations here, I think we are probably good to go.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
-## 2.2.0-nullsafety-dev
+## 3.0.0-nullsafety-dev
```
Comment (natebosch) [inline CHANGELOG.md]: If we are going to publish now.

```suggestion
## 3.0.0-nullsafety
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 
 Pre-release for the null safety migration of this package.
 
-Note that 2.2.0 may not be the final stable null safety release version, we
-reserve the right to release it as a 3.0.0 breaking change.
-
 This release will be pinned to only allow pre-release sdk versions starting from
-2.10.0-2.0.dev, which is the first version where this package will appear in the
+2.12.0-0, which is the first version where this package will appear in the
 null safety allow list.
```
Comment (natebosch) [inline CHANGELOG.md]: This entire paragraph can be removed. We are no longer pinning to pre-release, and the allow list isn't a factor in the `2.12.0` SDKs.

**Review by natebosch — APPROVED** [review]
Has this made it into the SDK and google3?

**Comment by natebosch** [comment]
> Has this made it into the SDK and google3?

It hasn't. https://dart-review.googlesource.com/c/sdk/+/170762

We can't publish to pub until after this lands and gets rolled.

**Comment by kevmoo** [comment]
I'm away from my keyboard for the rest of the day. If someone wants to take
over this pull request and land it. Feel free...

**Comment by jakemac53** [comment]
We won't have a roll internally until next week I assume so we should wait until then

File: CHANGELOG.md (hunk 2 above)
Comment (kevmoo) [reply CHANGELOG.md]: ack

**Comment by kevmoo** [comment]
> We won't have a roll internally until next week I assume so we should wait until then

ack. I'll land but we'll hold off publishing.

**Pull request merged**